database_connection.js

```diff
@@ -1,31 +1,23 @@
 const mysql = require("mysql2/promise");
 
-const query = async ()=>{
+
+
+const main = async (db_request)=>{
```

Comment:
You need to work on your variable names 😂 so `db_request` can just be `request` or even `req`